docs/sources/release-notes/upgrading-helm-from-2.1-to-3.0.md

```diff
@@ -0,0 +1,317 @@
+---
+title: "TODO: Upgrading to mimir-distributed 3.0"
+menuTitle: "TODO: Helm chart v3.0"
+description: "Upgrade guide for the Grafana Mimir Helm chart version 3.0"
+weight: 100
+---
+
+# Grafana Mimir Helm chart version 3.0 Upgrade Guide
+
+## Breaking Changes
+
+Several parameters available in version 2.1 of the mimir-distributed Helm chart have changed.
+Follow these steps to migrate your Helm values file to 3.0.
+
+- **Configuration Storage has changed**
+    - Mimir configuration is now stored in a ConfigMap by default instead of a Secret
+    - See [(Optional) Use the new, simplified configuration method](#optional-use-the-new-simplified-configuration-method) for tips on securing credentials in this new format
+    - The previous behavior can be enabled by setting `configStorageType` to `Secret`
+- **Update memcached configuration**
+    - The mimir-distributed chart supports four different caches. The configuration parameters have changed as follows:
+    - If you are not overriding the value of anything under the `memcached`, `memcached-queries`, `memcached-metadata`, or `memcached-results` sections, then you can safely skip this section.
```

Comment:
I think this is a bit misleading, I bet some people don't think of `enabled` as overriding stuff.

```suggestion
    - If you have not enabled any memcached caches,  and you are not overriding the value of anything under the `memcached`, `memcached-queries`, `memcached-metadata`, or `memcached-results` sections, then you can safely skip this section.
```